.github/workflows/ci.yml

```diff
@@ -5,12 +5,12 @@ on:
   pull_request:
     branches: [ master ]
 env:
-  CI_XCODE_VER: '/Applications/Xcode_11.7.app/Contents/Developer'
-  CI_XCODE_VER_OLDER: '/Applications/Xcode_11.6.app/Contents/Developer'
+  CI_XCODE_11: '/Applications/Xcode_11.7.app/Contents/Developer'
+  CI_XCODE_13: '/Applications/Xcode_13.0.app/Contents/Developer'
 
 jobs:
   ios:
-    runs-on: macos-latest
+    runs-on: macos-11
```

Comment:
You can switch these back to `macos-latest` as actions has now set macOS 11 to the default.

Reply:
👍🏼✅